.github/workflows/lint-and-test.yml

```diff
@@ -7,21 +7,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.0.2
```

Comment:
There were no breaking changes 👀 
release notes: https://github.com/actions/checkout/releases

Reply:
Any reason why you pin this to specific version rather than keep it to `v3`?

Reply:
No particular reason, whichever version is fine.
Shall I pin on v3 ?🙋

Reply:
I'd keep it to v3, which would also match what we had with v2.

Reply:
All right, I will fix it 🙋